packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/SimulationResource.java

```diff
@@ -27,7 +25,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.smallrye.reactive.messaging.annotations.Broadcast;
 import org.eclipse.microprofile.reactive.messaging.Emitter;
-import java.util.Map;
+import io.vertx.core.json.JsonObject;
```

Comment:
@dvince2 is this okay to use JsonObject instead of JsonNode?
I don't see why the former is better or necessary.

Reply:
Working with @Jami159 yesterday, the object being returned from here seems to be coming in as a `JsonObject` so we went with that - admittedly unsure why a `JsonNode` isn't being returned or what the difference here is.
Once we move to Spring I imagine this will end up as a JsonNode